shogun-config/src/main/resources/application-base.yml

```diff
@@ -13,12 +13,25 @@ spring:
   main:
     banner-mode: off
   jpa:
+    open-in-view: true
```

Comment:
I faintly remember this was some source of a performance penalty in the stone ages… I suspect we actually need thsi, correct?

Reply:
Please see [this article](https://www.baeldung.com/spring-open-session-in-view), and then choose an option.

Reply:
Thanks for the note and I think we should discuss the usage of the session in view in general. Actually the change in here is just the default for spring boot apps and hides a message on application startup.

Reply:
And yes, currently we need this setting. Otherwise the lazy initialized references must be fetched manually.